Extensions/Terraform/Src/Tasks/TerraformTaskV1/task.json

```diff
@@ -0,0 +1,318 @@
+{
+    "id": "FE504ACC-6115-40CB-89FF-191386B5E7BF",
+    "name": "TerraformCustomTask",
+    "friendlyName": "Custom Terraform Task",
+    "description": "Execute terraform cli commands",
+    "author": "Akash Chakrabarti",
+    "helpMarkDown": "",
+    "category": "Deploy",
+    "visibility": [
+        "Release"
+    ],
+    "demands": [
+        "terraform"
```

Comment:
I am not sure if we have plans to update the agent capability with terraform. How do you plan to handle the demands?